public/index.php

```diff
@@ -16,5 +16,25 @@
 $channel->publish("play", "meow");
 $channel->publish("play", "cluck");
 
+echo "Releasing channel...\n";
```

Comment:
I'm not sure I follow why you are releasing the channel here in this demo. Release in the REST lib doesn't do anything anyway as it's stateless.